devs/docs/es-backports.rst

```diff
@@ -386,19 +386,19 @@ components, so delaying it doesn't make applying other patches more difficult.
 - [ ] ef8f90cc2a5 Also mmap terms index (`.tip`) files for hybridfs (#43150)
 - [ ] 62620f28663 Remove usage of FileSwitchDirectory  (#42937)
 - [ ] 8159fdfcc9a Fix assertion in ReadOnlyEngine (#43010)
-- [ ] ca5771c1f4a Improve translog corruption detection (#42744)
-- [ ] ae96c16f88f Add a merge policy that prunes ID postings for soft-deleted but retained documents (#40741)
-- [ ] b6213ffa52d Revert "Add a merge policy that prunes ID postings for soft-deleted but retained documents (#40741)"
-- [ ] 186b52c5738 Add a merge policy that prunes ID postings for soft-deleted but retained documents (#40741)
+- [x] ca5771c1f4a Improve translog corruption detection (#42744)
+- [x] ae96c16f88f Add a merge policy that prunes ID postings for soft-deleted but retained documents (#40741)
+- [s] b6213ffa52d Revert "Add a merge policy that prunes ID postings for soft-deleted but retained documents (#40741)"
+- [s] 186b52c5738 Add a merge policy that prunes ID postings for soft-deleted but retained documents (#40741)
 - [x] 8ffb5a15d22 Fix Infinite Loops in ExceptionsHelper#unwrap (#42716)
-- [ ] 4dd5be8e3f3 Use reader attributes to control term dict memory useage (#42838)
-- [ ] 56a662ed288 Remove Support for VERSION_CHECKPOINTS Translogs (#42782)
-- [ ] 8e5812c4b32 remove v6.5.x and v6.6.x version constants (#42130)
-- [ ] 6e39433cd53 Remove "nodes/0" folder prefix from data path (#42489)
-- [ ] f07b90f3c35 Remove support for chained multi-fields. (#42333)
-- [ ] 4a08b3d1c94 remove 6.4.x version constants (#42127)
-- [ ] 1ef00e368b4 Avoid loading retention leases while writing them (#42620)
-- [ ] 2c8440922f9 Remove PRE_60_NODE_CHECKPOINT (#42527)
+- [s] 4dd5be8e3f3 Use reader attributes to control term dict memory useage (#42838)
+- [d] 56a662ed288 Remove Support for VERSION_CHECKPOINTS Translogs (#42782)
```

Comment:
If I'm not mistaken we need to wait for 5.0 for this, we may have people updating from 3.0 to 4.3 (although not recommended)